templates/content/node--oe-project--oe-w-content-banner.html.twig

```diff
@@ -0,0 +1,17 @@
+{#
+/**
+ * @file
+ * Project content banner display.
+ */
+#}
+{{ pattern('content_banner', {
+  'background': 'gray',
+  'title': label,
+  'content': content.oe_teaser,
+  'image': image,
+  'attributes': create_attribute().addClass(['ps-0']),
+  'badges': content.oe_subject ? content.oe_subject|field_value|default([])|map(_item => {
```

Comment:
next level chaining of filters, but a bit hard 2 read, can we make this more readable for maintainability?

Reply:
would be more readable if we had some custom twig filters.

Reply:
All the parts are still needed to deal with non-array values. And I think this stuff is still better than foreach + merge.
I can change the order, I can insert line breaks.
E.g.
```
  'badges': content.oe_subject ? (content.oe_subject
    |field_value
    |default([])
    |filter(_item => _item|drupal_escape|striptags|trim is not empty)
    |map(_item => {
      'label': _item,
      'rounded_pill': true,
    }))
```

Reply:
or
```
  'badges': content.oe_subject ? (content.oe_subject
    |field_value|default([])
    |filter(_item => _item|drupal_escape|striptags|trim is not empty)
    |map(_item => {
      'label': _item,
      'rounded_pill': true,
    }))

Reply:
Actually we can drop the first ternary, because `|field_value` allows null passed to it. The rest will just process an empty array which is ok.